src/test/java/edu/harvard/iq/dataverse/dataaccess/StorageIOTest.java

```diff
@@ -65,7 +65,9 @@ public void testGetDvObject() {
         Dataset d= new Dataset();
```

Comment:
It would be great to have more tests in this pull request. It's lots of new code.